content/study/posts/liberia_custom-mb.json

```
@@ -0,0 +1,60 @@
+{
+  "version": 8,
+  "sources": {
+    "transmission_lines_1": {
+      "data": [
+        "https://energydata.info/dataset/AICD-AFR-62/resource/029caf85-1dca-4a76-b51b-54dbe51b3bf8/download/liberia-electricity-transmission-network.geojson"
+      ],
```

```suggestion
      "data": "https://energydata.info/dataset/AICD-AFR-62/resource/029caf85-1dca-4a76-b51b-54dbe51b3bf8/download/liberia-electricity-transmission-network.geojson",
```
As soon as the `data` property is defined as an array, mapbox interprets this as inline geojson, reading this as an array of 1 string of text, which is invalid.

The url should be used directly after `data`. Exactly as it is being done for the `population` source.
Docs for the geojson sources are available at: https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#geojson

---

Ok, got it. But still when I am running locally I am not able to render the geojson on the map. The [source](https://energydata.info/dataset/AICD-AFR-62/resource/029caf85-1dca-4a76-b51b-54dbe51b3bf8/download/liberia-electricity-transmission-network.geojson) seems to be correct.